### ISSUE TITLE
What's the `native-image` command on Windows?

### ISSUE THREAD

**Issue by guizmaii** [graalvm/setup-graalvm#50]
I have a CI defined this way:
```yml
jobs:
  native-image:
    name: Compile
    runs-on: ${{ matrix.os }}
    strategy:
      fail-fast: false
      matrix:
        os: [macos-latest, ubuntu-latest, windows-latest]
        java: ['17.0.7', '20.0.1']
        distribution: ['graalvm', 'graalvm-community']
    steps:
      - uses: actions/checkout@v3
      - uses: graalvm/setup-graalvm@v1
        with:
          java-version: ${{ matrix.java }}
          distribution: ${{ matrix.distribution }}
          github-token: ${{ secrets.GITHUB_TOKEN }}
          cache: sbt
          components: native-image
      - name: sbt test
        shell: bash
        if: ${{ matrix.os == 'windows-latest' }}
        run: |
          # Copied from https://github.com/graalvm/setup-graalvm#quickstart-template
          echo "GRAALVM_HOME: $GRAALVM_HOME"
          echo "JAVA_HOME: $JAVA_HOME"
          java --version
          native-image --version 
          sbt example/nativeImage
```
but the execution fails when trying to execute the `native-image --version` line with this error:
```bash
D:\a\_temp\1269dfb9-eef7-4ee6-af57-30494186cd6e.sh: line 5: native-image: command not found
Error: Process completed with exit code 127.
```

Details: https://github.com/scalameta/sbt-native-image/actions/runs/5568848315/jobs/10171775833

**Comment by ozkanpakdil** [comment]
native-image was not included in 17. [here](https://www.graalvm.org/22.0/reference-manual/native-image/#install-native-image) says you can install it with command below
```shell
gu install native-image
```

**Comment by fniephaus** [comment]
> but the execution fails when trying to execute the `native-image --version` line with this error:

You've set `shell: bash` and it seems that `native-image` is not resolved to `native-image.cmd` on Windows. This automatically happens in a PowerShell (the default shell on Windows) or a Command Prompt.
Could you try running `native-image.cmd --version` on Windows or use a different shell?

> native-image was not included in 17

I'm not sure this is accurate, `native-image` is included in the `17.0.7` release of GraalVM.

**Comment by guizmaii** [comment]
> Could you try running `native-image.cmd --version` on Windows or use a different shell?

This is working. Thanks :)
